### ISSUE TITLE
[Data@TAMU] Search Page

### ISSUE THREAD

**Issue by devangm** [TAMULib/data-dspace-angular#16]
**URL:** https://data-pre.library.tamu.edu/search?spc.page=1&spc.sf=dc.date.accessioned&spc.sd=DESC
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.3.1 Info and Relationships (A)
**Issues:** Two H1 designations
**Notes:** Both "Data@TAMU" and "Search Results" are labeled H1

**Comment by tamu-sad-iii** [comment]
I am unable to find the Two H1 designations on the provided URL.

<img width="1920" height="1008" alt="Image" src="https://github.com/user-attachments/assets/e1e71892-08c0-42f3-9145-4f8a362fe419" />
